contracts/RLPReader.sol

```diff
@@ -29,6 +55,19 @@ library RLPReader {
         return RLPItem(item.length, memPtr);
     }
 
+    /*
+    * @dev Create an iterator.
+    * @param self The RLP item.
```

Comment:
it would be nice to note that this will revert if the RLPItem is not of a list

Reply:
Added the note. Also added some additional docs for the other iterator functions.